docs/man/coreos-installer-install.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+parent: Command line reference
+nav_order: 1
+---
+
+# coreos-installer install
+
+## Description
+
+Install Fedora CoreOS or RHEL CoreOS
+
+## Usage
+
+* coreos-installer install [OPTIONS] &lt;device&gt;
+
+## Options
```

Comment:
Several of these will need a lot more text, but that can be left for a followup.